### ISSUE TITLE
[Bug]: Astro Einstellungen wieder nicht erreichbar (wie bei #1578)

### ISSUE THREAD

**Issue by winnyschuster** [ioBroker/ioBroker.javascript#1605]
### I'm sure that

- [X] This issue is still present in the **current beta version** of this adapter
- [X] There is no other (open) issue with the same topic (use the search!)
- [X] This issue is not described in the adapter documentation / FAQ (read the docs!)

### Script type

N/A

### The problem

seit der v8.5.0 sind die Astro Einstellungen wieder nicht erreichbar, wie bei #1578. Admin ist auf 6.17.14. Folgende Datei bringt einen 404 Error: http://localhost:8081/adapter/javascript/custom/static/js/vendors-node_modules_mui_utils_visuallyHidden_visuallyHidden_js-node_modules_popperjs_core_li-ba9263.119cfc7b.chunk.js

### iobroker.current.log (in debug mode!)

N/A

### Version of nodejs

20.14.0

### Version of ioBroker js-controller

5.0.19

### Version of adapter

8.5.1

**Comment by klein0r** [comment]
Warum kommen denn ständig neue Dateien hinzu... 😞 Es hat sich nix geändert (nur die Dependency zu adapter-react-v5 erhöht)

**Comment by winnyschuster** [comment]
ist mit der 8.5.2 wieder O.K, sehr komisch

**Comment by klein0r** [comment]
8.4.2 wurde mit adapter-react-v5 4.13.24 gebaut:

https://github.com/ioBroker/ioBroker.javascript/blob/e04c53c85f3f3cea1e583f2fbb3bad657798517e/src-admin/package.json#L14C41-L14C48

8.5.2 wurde mit adapter-react-v5 5.0.4 gebaut:

https://github.com/ioBroker/ioBroker.javascript/blob/97a5cdd44f7738b18f3f5e32b96aee384f9cbb54/src-admin/package.json#L14C1-L15C1

Der Build Process nimmt aber immer die aktuellste Version (weil package-lock nicht eingecheckt) und daher wahrscheinlich mit 5.0.5 gebaut.

**Comment by winnyschuster** [comment]
8.5.2 ist ja wieder ok, betroffen waren nur 8.5.0 + 8.5.1. Hauptsache du hast keine Arbeit damit :-) Hatte die neueste zu spät gesehen, da war das issue schon raus :-)

**Comment by klein0r** [comment]
> 8.5.2 ist ja wieder ok

Das macht es für mich eher noch undurchsichtiger, ...

**Comment by winnyschuster** [comment]
ja, ich blicke da auch nicht durch, habe es aber nachvollziehbar unter WIN und WSL getestet, auch dass es mit 8.5.2 dann wieder funktioniert. 

BTW: Habe den [Kommentar hier](https://github.com/ioBroker/ioBroker.javascript/blob/97a5cdd44f7738b18f3f5e32b96aee384f9cbb54/lib/sandbox.js#L1698) gesehen. getAstroStartOfDay() funktioniert nicht über alle Zeitzonen hinweg und suncalc liefert dann den Tag zuvor. Teste mal zB mit Perth(lat: -31.95289, lng: 115.85152). Ich hatte das seinerzeit über alle Kontinente und Zeitzonen getestet, mit 12Uhr lokaler Zeit (nicht UTC!) liefert suncalc immer die Werte für den  derzeitigen Tag
